dll/cpl/desk/screensaver.c

```diff
@@ -308,6 +304,8 @@ ScreensaverConfig(HWND hwndDlg, PDATA pData)
 
         if (WaitForSettingsDialog(hwndDlg, pi.hProcess))
             SetScreenSaverPreviewBox(hwndDlg, pData);
+        CloseHandle(pi.hProcess);
+        CloseHandle(pi.hThread);
```

Comment:
Minor code style suggestion, please add a newline before these:
```suggestion

        CloseHandle(pi.hProcess);
        CloseHandle(pi.hThread);
```

Reply:
Done